pkg/agent/cniserver/interface_configuration_linux_test.go

```diff
@@ -300,52 +300,46 @@ func TestAdvertiseContainerAddr(t *testing.T) {
 			name:    "interface-not-found",
 			runInNS: true,
 			result: &current.Result{IPs: []*current.IPConfig{
-				{Version: "4", Interface: &interfaceID, Address: ipv4CIDR, Gateway: ipv4Gateway},
-				{Version: "6", Interface: &interfaceID, Address: ipv6CIDR, Gateway: ipv6Gateway},
+				{Interface: &interfaceID, Address: ipv4CIDR, Gateway: ipv4Gateway},
```

Comment:
So the `Version` is no longer required in cni v1.0? I am wondering will this work properly with old plugins? I mean the cni plugins which is installed by `build/image/scripts/install_cni`

Reply:
Yes, `Version` property is not existing in the Result type since 1.1. I have added function to convert the Result in CNI Response to the previous version format if antrea CNI configuration not using the latest version.